src/homework2.sql

```diff
@@ -0,0 +1,70 @@
+USE baseball;
+
+DROP TABLE IF EXISTS batter_game_stat;
+
+CREATE TABLE batter_game_stat AS
+SELECT
+    bc.batter AS batter_id,
+    bc.game_id,
+    DATE(game.local_date) AS game_date,
+    bc.atBat AS AB,
+    bc.Hit AS hit
+FROM batter_counts bc
+JOIN game ON bc.game_id = game.game_id
+WHERE bc.atBat > 0;
```

Comment:
This line helps avoiding error of division by zero; it's really smart that you thought about this ahead of the time.